Tools/Scripts/webkitpy/w3c/test_importer.py

```diff
@@ -212,6 +212,14 @@ def __init__(self, port, test_paths, options):
     def do_import(self):
         if self.source_directory:
             source_path = str(Path(self.source_directory) / 'web-platform-tests')
+            if not Path(source_path).exists():
+                _log.error("There isn't a web-platform-tests directory in the path you have provided: %s" % source_path)
+                exit(1)
```

Comment:
Can we not exit the entire Python process, but instead just return from the function? We don't want random classes and functions stopping the entire execution flow, especially when they're sometimes used as a library. See the other error cases below.

Reply:
So yes this is the discussion we had with @marcoscaceres
The `return` is a bit annoying here, because it continues to process things which are useless or sometimes can be destructive.  

Like this case:
`import-w3c-tests web-platform-tests/digital-whatever --src-dir /tmp`
Where it will even modify json file, without anything valid. 

But indeed if this is used by another library. `do_import` is too big :/